src/Components/ProductList/Paginator.js

```diff
@@ -4,32 +4,31 @@ import { PaginatorContainer } from 'Components/ProductList/styled';
 import rightArrow from 'assets/svgs/right-arrow.svg';
 import leftArrow from 'assets/svgs/left-arrow.svg';
 
-const Paginator = ( props ) => {
+const Paginator = ( { numberOfPages, next, prev, state, setStater } ) => {
+    
+    const Pages = Array.from(Array(numberOfPages).keys());
```

Comment:
This should be camelCased. Initially I thought this was a component because of the PascalCased name.